chapter_deep-learning-computation/parameters_vn.md

```diff
@@ -200,11 +203,10 @@ It has the same shape as the weight.
 Because we have not invoked backpropagation for this network yet, its values are all 0.
 -->
 
-Chú ý rằng các trọng số là khác không.
-Điều này là có chủ ý vì chúng được khởi tạo ngẫu nhiên khi ta xây dựng mạng.
-`data` không phải là hàm duy nhất mà ta có thể gọi.
-Chẳng hạn, ta có thể gọi hàm truy cập gradient theo các tham số.
-Các gradient này có cùng kích thước với trọng số, tuy nhiên tất cả các giá trị hiện tại đều bằng 0 bởi vì ta chưa gọi hàm lan truyền ngược.
+Chú ý rằng khác với hệ số điều chỉnh, trọng số chứa các giá trị khác không bởi chúng được khởi tạo ngẫu nhiên.
+Ngoài `data`, mỗi `Parameter` còn cung cấp phương thức `grad()` để truy cập gradient.
+Gradient sẽ có cùng kích thước với trọng số. <!-- thế còn bias thì sao ._. -->
```

Comment:
2 câu cuối như kiểu cho đoạn mã bên dưới thôi ý nên mới kêu grad==weight.

Reply:
ah đúng rồi, thanks. Chả hiểu sao dạo này dịch cứ toàn bỏ qua context